dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/CrudControllerAdvice.java

```diff
@@ -150,6 +152,13 @@ public CrudControllerAdvice() {
 
   @InitBinder
   protected void initBinder(WebDataBinder binder) {
+    binder.registerCustomEditor(
```

Comment:
I wanted to talk to you about Spring binding as I learned something while you were out for a bit. See https://github.com/dhis2/dhis2-core/pull/16673

I think we can simplify this. Our type needs to have a constructor or valueOf factory to turn a String into a Start/EndDate. We then do not need to register anything and rely on Springs conversion system. See PR description on how it works.

Reply:
Yesss.
I was confused by the code as it wasn't really as I remembered :grimacing: 
Changed to use Springs conversion